www/src/data/sidebars/tutorial-links.yaml

```diff
@@ -71,7 +71,7 @@
         - title: What are transformer plugins?
           link: /tutorial/part-six/#transformer-plugins
         - title: Create a list of our site's markdown files
-          link: /tutorial/part-six/#create-a-list-of-our-sites-markdown-files-in-srcpagesindexjs
+          link: /tutorial/part-six/#create-a-list-of-your-sites-markdown-files-in-srcpagesindexjs
```

Comment:
The hash should be `create-a-list-of-your-sites-markdown-files-in-code-classlanguage-textsrcpagesindexjscode`

Here is the html generated on that header element:
```
<a href="#create-a-list-of-your-sites-markdown-files-in-code-classlanguage-textsrcpagesindexjscode" aria-hidden="true" class="anchor"></a>
```

Reply:
hrmmm we need to fix whatever is causing the language codes to be added to the hashes.

Reply:
I just made a commit with ```create-a-list-of-your-sites-markdown-files-in-code-classlanguage-textsrcpagesindexjscode``` instead the first change I initially made.

Reply:
@KyleAMathews langeuage codes are added due to backticks in h2 for filename.

Shall we investigate it as separate issue?

Reply:
PR that solves backticks problem in header ids - https://github.com/gatsbyjs/gatsby/pull/6856